facdb/sql/dsny_electronicsdrop.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS _dsny_electronicsdrop;
+SELECT uid,
+    source,
+    CONCAT(dropoff_sitename, ' ', 'Electronics Drop-off Site') as facname,
```

Comment:
Question - might need LS to weigh in.  Should the facname be `(dropoff_sitename, ': DSNY Electronics Drop-off Site')`?   Applies to other new DSNY sources.

Reply:
Keep as is (i.e. Best Buy Electronics Drop-off Site)